modules/default/weather/weather.js

```diff
@@ -27,6 +27,7 @@ Module.register("weather", {
 		showPeriodUpper: false,
 		showPrecipitationAmount: false,
 		showPrecipitationProbability: false,
+		showUV_Index: false,
```

Comment:
Could you rename that config option to showUVIndex (so it stays consistent with the other options)?

Reply:
done